src/implot_visualization/app/CMakeLists.txt

```diff
@@ -1,10 +1,11 @@
-target_sources(index PRIVATE main.cpp)
+target_sources(index PRIVATE main.cpp deserialize_json.cpp emscripten_fetch.cpp)
 
```

Comment:
I think before Merging, we should maybe futher improve/structure that. E.g. create library targets.